contracts/src/interfaces/IAllowlist.1.sol

```diff
@@ -68,9 +86,22 @@ interface IAllowlistV1 {
     /// @param _newAllowerAddress New address allowed to edit the allowlist
     function setAllower(address _newAllowerAddress) external;
 
-    /// @notice Sets the allowlisting status for one or more accounts
-    /// @dev The permission value is overridden and not updated
-    /// @param _accounts Accounts with statuses to edit
-    /// @param _permissions Allowlist permissions for each account, in the same order as _accounts
-    function allow(address[] calldata _accounts, uint256[] calldata _permissions) external;
+    /// @notice Changes the denier address
+    /// @param _newDenierAddress New address allowed to edit the allowlist
+    function setDenier(address _newDenierAddress) external;
+
+    /// @notice Sets the allow permissions for one or more accounts
+    /// @dev This function is for allocating or removing deposit, redeem or donate permissions.
+    ///      This function could be used to give any permissions that we come up with in the future.
+    ///      An address which was denied has to be undenied first before they could be given any permission(s).
+    /// @param _accounts Accounts to update
+    /// @param _permissions New permission values
+    function setAllowPermissions(address[] calldata _accounts, uint256[] calldata _permissions) external;
```

Comment:
fwiw, i am still undecided about what best to do here, if we keep this function as `allow` then we don't break any APIs, we are just adding new ones in ... thoughts ? happy to talk through the tradeoffs here.

Reply:
as discussed, let's stick with the naming in the PR, it better represents what the functions actually do